chore: bump cephfs_interfaces

jedel1043 opened the pull request:
Also adds a test for the new behaviour, which doesn't error after reintegrating with a cephfs client.

## Checklist

- [x] I am the author of these changes, or I have the rights to submit them.
- [x] I have added the relevant changes to the README and/or documentation.
- [x] I have self reviewed my own code.
- [x] All requested changes and/or review comments have been resolved.

jedel1043 committed:
chore: bump cephfs_interfaces

Adds a test for the new behaviour, which doesn't error after reintegrating with a cephfs client.

NucciTheBoss reviewed (approved):
LGTM :rocket:

NucciTheBoss commented:
Hmm... seems like failed test is unhappy about the http+unix socket used to communicate with the LXD REST API. Let me try re-running the action to see if this was just an intermittent failure.

jedel1043 committed:
fix: pin requests to make pylxd work

Track https://github.com/psf/requests/issues/6707 to ensure we remove the pin after the issue is fixed.
